new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -1660,8 +1710,223 @@ public static Path generateFileFromTemplate(
     String out = targetFile.toString();
     for (Map.Entry<String, String> entry : templateMap.entrySet()) {
       logger.info("Replacing String {0} with the value {1}", entry.getKey(), entry.getValue());
-      replaceStringInFile(out, entry.getKey(), entry.getValue()); 
+      replaceStringInFile(out, entry.getKey(), entry.getValue());
     }
     return targetFile;
   }
+
+  /**
+  * Create a persistent volume.
+   *
+   * @param pvName name of the persistent volume to create
+   * @param domainUid domain UID
+   * @param className name of the class to call this method
+   */
+  public static void createPV(String pvName, String domainUid, String className) {
```

Comment:
You need to get logger in each method it looks like.

LoggingFacade logger = getLogger();

Reply:
Done